game/games/sortit/box.gd

```diff
@@ -14,6 +14,21 @@ export(float) var number_scale = 0.8
 
 var number: int = 0
 var is_black: bool = false
+var force_sleep: bool = false
+
+
+func _process(_delta):
+	# Handle boxes falling through the ground (Should not happen)
+	if translation.y < -10:
+		push_warning("SortIt: Box dropped through the ground")
+		_on_despawn_timer_timeout()
```

Comment:
Here, move the logic inside `_on_despawn_timer_timeout()` into its own function, like `despawn()` and call that instead (same inside the signal callback). I'd consider it bad practice to call such callback functions by hand, although it's certainly vlaid.

Reply:
I agree.